app/components/slider-control.js

```diff
@@ -0,0 +1,61 @@
+import Ember from 'ember';
+import layout from '../templates/components/slider-control';
+
+export default Ember.Component.extend({
+  layout: layout,
+  classNames: ['slider-control'],
+  classNameBindings: ['isMouseDown:dragging'],
+  percentage: 0,
+
+  setupExternalListeners: function() {
+    function resize (that) {
+      that.set("width", that.element.clientWidth);
+    }
+
+    $(window).on("resize", () => resize(this));
+    resize(this);
+
+    $(window).on("mousemove", (e) => { this.mousemove(e, this); });
```

Comment:
These listeners are needed so the slider works when dragging starting from inside the slider to outside the slider.